src/client.js

```diff
@@ -1,5 +1,5 @@
 var _ = require("lodash");
-var package = require("../package.json");
+var pkg = require("../package.json");
```

Comment:
Also, if you add `"use strict";` at the top (and feel free to do so on files you changed as it's a really good thing to have), [`implements`, `interface`, `let`, `package`, `private`, `protected`, `public`, `static`, and `yield` become reserved keywords](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode#Paving_the_way_for_future_ECMAScript_versions).